spec/oauth2/client_spec.rb

```diff
@@ -159,6 +159,23 @@
     end
   end
 
+  describe '#connection' do
+    it 'smoothly handles succeeding requests when OAUTH_DEBUG=true' do
+      env = double('ENV')
+      allow(env).to receive(:[]).with('OAUTH_DEBUG').and_return('true')
+
+      capture_output do
+        # first request (always goes smoothly)
+        subject.request(:get, '/success')
+      end
+
+      expect do
+        # second request (used to throw Faraday::RackBuilder::StackLocked)
+        subject.request(:get, '/success')
+      end.not_to raise_error
```

Comment:
Need `end.not_to raise Faraday::RackBuilder::StackLocked` here. `raise_error` doesn't fail on `Faraday::RackBuilder::StackLocked` for some reason.